bluebrain/repo-bluebrain/packages/py-neurodamus/package.py

```diff
@@ -8,8 +8,8 @@
 class PyNeurodamus(PythonPackage):
     """The BBP simulation control suite, Python API"""
 
-    homepage = "https://bbpteam.epfl.ch/project/spaces/display/BGLIB/Neurodamus"
-    git = "ssh://git@bbpgitlab.epfl.ch/hpc/sim/neurodamus-py.git"
+    homepage = "https://github.com/BlueBrain/neurodamus"
+    git = "ssh://git@bbpgitlab.epfl.ch/hpc/sim/neurodamus.git"
```

Comment:
Any particular reason why you're not pointing to the github repository for the git attribute?

Reply:
I wasn't sure how the CI is going to handle this with the permissions set to the github repo when cloning it. Also I'm not sure whether all tags, branches, etc are transferred there

Reply:
It's public, no? Then we should actually use `http` access!